packages/anoncreds/src/utils/credential.ts

```diff
@@ -1,7 +1,7 @@
 import type { AnonCredsSchema, AnonCredsCredentialValues } from '../models'
 import type { CredentialPreviewAttributeOptions, LinkedAttachment } from '@aries-framework/core'
 
-import { AriesFrameworkError, Hasher, encodeAttachment } from '@aries-framework/core'
+import { AriesFrameworkError, Hasher, encodeAttachment, Buffer } from '@aries-framework/core'
```

Comment:
Hmm I added an eslint rule to disallow global Buffer. Not sure why it doesn't work?...